src/app/ngrx/reducers/index.ts

```diff
@@ -76,6 +76,18 @@ export const selectEpisodesError = createSelector(selectEpisodeState, getEpisode
 export const selectSelectedPageEpisodes = createSelector(selectEpisodes, selectPageRoute, (episodes: EpisodeModel[], page: number) => {
   return episodes.filter(episode => episode.page === page);
 });
+export const selectMostRecentEpisode = createSelector(selectSelectedPodcast, selectEpisodes, (pod: PodcastModel, eps: EpisodeModel[]) => {
+  // TODO: if you didn't start on the first page, this isn't the most recent
```

Comment:
Any ideas on this one?  If the route starts on `page=2`, you'll never know about the most recent 10 episodes.  Bummer!

Maybe the next PR on this (to pull in the all-time-metrics) should just have an independent CMS call to get the 1-most-recent episode?